tests/docker/Dockerfile

```diff
@@ -135,6 +135,10 @@ COPY --chown=0:0 tests/setup.py /root/tests/
 RUN python3 -m pip install --upgrade --force pip && \
     python3 -m pip install --force --no-cache-dir -e /root/tests/
 
+RUN git -C /opt clone https://github.com/redpanda-data/openmessaging-benchmark.git && \
```

Comment:
kubernetes option should be turned on as well.

Reply:
I will add clustered ducktape support later.
We will also need to add time sinchronization to clustered ducktape to run OMB